packages/main/src/RPA/Desktop/keywords/finder.py

```diff
@@ -10,6 +10,8 @@
     parse_locator,
 )
 
+from RPA.Desktop.keywords.screen import _get_displays, _region_from_mss_monitor
```

Comment:
if these are used outside the `screen` module, they probably should not prefixed with underscores